bitnami/etcd/3.5/debian-12/rootfs/opt/bitnami/scripts/libetcd.sh

```diff
@@ -307,7 +307,8 @@ etcdctl_auth_norbac_flags() {
         authFlags+=("--cert" "${ETCD_DATA_DIR}/fixtures/client/cert.pem" "--key" "${ETCD_DATA_DIR}/fixtures/client/key.pem")
     else
         [[ -f "$ETCD_CERT_FILE" ]] && [[ -f "$ETCD_KEY_FILE" ]] && authFlags+=("--cert" "$ETCD_CERT_FILE" "--key" "$ETCD_KEY_FILE")
-        [[ -f "$ETCD_TRUSTED_CA_FILE" ]] && authFlags+=("--cacert" "$ETCD_TRUSTED_CA_FILE")
+        # if CA file exists, then use CA to verify server certs; otherwise, just skip server certs verification
+        [[ -f "$ETCD_TRUSTED_CA_FILE" ]] && authFlags+=("--cacert" "$ETCD_TRUSTED_CA_FILE") || authFlags+=("--insecure-skip-tls-verify")
     fi
```

Comment:
I feel like there is some redundancy between setting `--insecure-skip-tls-verify` here and in the `healthcheck.sh` script. Do you think it can be simplified somehow?

For example, removing it from `healthcheck.sh` and apply the following change:

```suggestion
    fi
    # If CA file exists, then use CA to verify server certs; otherwise, just skip server certs verification
    [[ -f "$ETCD_TRUSTED_CA_FILE" ]] && authFlags+=("--cacert" "$ETCD_TRUSTED_CA_FILE") || authFlags+=("--insecure-skip-tls-verify")
    fi
```

But that could be insecure. Users not setting the recommended CA file would not notice it is missing, although it is recommended.

The other option is that you only set the flag in the `healthcheck.sh` script, changing the condition in which it is set so it fits you use case.